src/doc/api/api_base.nit

```diff
@@ -225,6 +225,10 @@ redef class MVirtualType
 	redef var api_url = mproperty.api_url is lazy
 end
 
+redef class HtmlightVisitor
```

Comment:
It is more clean to create a subclass instead of refining it.
Eventually you might like having a specific subclass for other kind of customization.